src/RotateObject3D.tsx

```diff
@@ -49,15 +50,24 @@ const RotateObject3D = () => {
       }
     );
 
-        
-    function handleMouseMove(event: MouseEvent) {
-      const mouseX = (event.clientX / window.innerWidth) * mouseXScaleFactor + mouseXOset;
-      rotationSpeed = mouseX * rotationSpeedFactor;
+    /*  
+    function handleMovement(event: MouseEvent) {
+      setRotationSpeed();
     }
 
-    document.addEventListener('mousemove', handleMouseMove);
+    document.addEventListener('mousemove', handleMovement);
+    */
+
+    function setRotationSpeed() {
+      if (Coordinates.isSet) {
+        //const mouseX = (event.clientX / window.innerWidth) * mouseXScaleFactor + mouseXOset;
```

Comment:
Please remove useless comment. If it is code that will be used in the future, then describe it appropriately.

Reply:
@Quarol  It is still not resolved.